### DISCUSSION TITLE
SQL syntax highlighting in 3.3.0?

### DISCUSSION THREAD

**Discussion by tedstriker2** [pawelsalawa/sqlitestudio#3986] [Q&A] [answered]
The highlighting is configurable according to [the user manual](https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#typing-sql-queries):

> When you type SQL query in the editor, it will highlight SQL syntax, but also if you type the correct name of any object (that is existing table, index, trigger, view, database or column name), the name gets highlighted with blue color (by default, it's configurable)

I can't seem to find how to change the keyword color, comment italics, etc. in version 3.3.0 - am I missing something?

**Comment by pawelsalawa** [comment, marked as answer]
Color configuration was removed, as it was introducing conflicts with dark themes. Perhaps I will think about a way to bring back configuration of syntax highlighting colors, but I need to rethink how to do it right, so it doesn't mess with bright/dark themes.

**Reply by tedstriker2** [reply]
Thanks for the quick reply. I understand making syntax colors compatible with dark themes is tricky.
